dask/dataframe/io/tests/test_parquet.py

```diff
@@ -967,6 +979,9 @@ def test_categories_unnamed_index(tmpdir, engine):
     if engine.startswith("pyarrow") and pa.__version__ < LooseVersion("0.15.0"):
         pytest.skip("PyArrow>=0.15 Required.")
 
+    if engine == "fastparquet" and PANDAS_GT_123:
+        pytest.skip("https://github.com/dask/fastparquet/issues/576")
```

Comment:
This shouldn't be needed anymore, I think, as I reverted the change in pandas (`Categorical._codes` is writeable again)

Reply:
Oh right! I should have made that an xfail so that I noticed it.